naga/tests/in/spv/unnamed-gl-per-vertex.spvasm

```diff
@@ -0,0 +1,72 @@
+;; Make sure that we don't have a validation error due to lacking capabilities
```

Comment:
Trying to mimic the existing `.spvasm` files, I manually created this file with `spirv-dis unnamed-gl-per-vertex.spv > unnamed-gl-per-vertex.spvasm` and added some relevant comments in it. I don't know if there are any standard practices for adding naga tests here that I might have missed.

Reply:
The tests look great, thanks for adding the comments as well!